keps/sig-release/20200122-kubernetes-yearly-support-period.md

```diff
@@ -0,0 +1,318 @@
+It is recognized that this is currently a pain point in the project and this KEP incrementally worsens the user experience for folks who defer to the maximum their upgrades.
+
+#### Maturity levels (alpha, beta, stable)
+
+Criteria for promotion from alpha to beta and beta to stable remain unchanged.
+
+#### Deprecation policy
+
+Unchanged from existing Kubernetes project deprecation criteria.  Deprecations for GA user interfaces today are already at least 12 months.
+
+### Version Skew Strategy
+
+Incremental change elongating existing Kubernetes project version skew policy, i.e.: where “N+/-1” to “N+/-2” and where “N+/-2” to “N+/-3”.
+This matters, especially for external clients programmatically interacting with a long-lived cluster, beyond the cluster control plane components, moved through the upgrade process.
+
+## Implementation History
+
+Major milestones in the life cycle of a KEP should be tracked in `Implementation History`.
+Major milestones might include
+
+- the `Summary` and `Motivation` sections being merged signaling SIG acceptance
+- the `Proposal` section being merged signaling agreement on a proposed design
+- the date implementation started
+- the first Kubernetes release where an initial version of the KEP was available
+- the version of Kubernetes where the KEP graduated to general availability
+- when the KEP was retired or superseded
+
+## Drawbacks
+
+The proposed change may encourage end-users to provide less frequent experimental feedback (eg: alpha/beta feature feedback) to the community on new releases, upgrade-ability, and compatibility.
+This is an existing problem in the project today for users who do not regularly update, it remains unaddressed by this KEP, but is not notably exacerbated.
+Users who did not upgrade regularly do not give us early feedback.
+Improving this remains as future work for the project.
+
+There is a risk this KEP is a slippery slope to subsequent iterative support extensions.
+There is a finite limit to the resources an open source community can give to supporting a given release and this KEP represents a pragmatic balance for our Kubernetes community.
+Discussion has shown there is zero community support for further elongation of the community support window beyond that proposed in this KEP.
+There is already active community resistance to the idea that one might attempt to further elongate support at the community level.
+Distributors/vendors remain free to carve out their own support niche if they feel there is a market for that.
+
+An increase to the support period of Kubernetes also increases the human burden of maintenance on the patch management (Release Engineering), test-infra, k8s-infra, sig-pm/release, and regular SIG functioning.
+Some of these impacts have already been recognized and sub-projects have been set up to mitigate them.
+Still, there will be an unavoidable increased cost in dollars for infrastructure resources kept online 3 additional months.
```

Comment:
in addition to engineering costs patching critical bugs in more versions.

Reply:
Fair enough, I'll add this.